grammar.js

```diff
@@ -175,6 +176,7 @@ module.exports = grammar({
       $.module_declaration,
       $.external_declaration,
       $.exception_declaration,
+      $.call_expression,
```

Comment:
Call _expression_ is definitely an _expression_, not a _declaration_.

I understand the intent, but it should be structured in another way I guess.

Reply:
I moved to `call_expression` to `_decorated_statement`